config/checkstyle/suppressions.xml

```diff
@@ -4,5 +4,8 @@
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
-  <!--add any suppressions here -->
+  <suppress checks=".*" files=".*Test.java"/>
+  <suppress checks=".*" files=".*[\\/]unit[\\/]"/>
+  <suppress checks="JavadocVariable" files="MicrosoftStsIdToken.java"/>
```

Comment:
What's this?

Reply:
Could not locate any documentation for these two variables.
`public static final String AIO = "aio";
    public static final String UTI = "uti";`